src/main.js

```diff
@@ -161,13 +171,28 @@ createBookBtn.addEventListener("click", function(event){
 });
 
 function makeBookButton() {
-  currentCover = createCover(userCover.value, userTitle.value, userDecr1.value, userDecr2.value);
-  coverImage.src = currentCover.coverImg; 
-  coverTitle.innerHTML = currentCover.title;
-  coverDescriptor1.innerHTML = currentCover.tagline1;
-  coverDescriptor2.innerHTML = currentCover.tagline2;
-  saveNewInfo();
-  goHome();
+  try {
+    new URL(userCover.value)
+  }
+  catch(error){
+    alert('Please enter a valid url‼️')
+    return
+  }
+  if (userCover.value === '' ||
+    userTitle.value === '' ||
+    userDecr1.value === ''||
+    userDecr2.value === '') {
+      alert('⚠️Please fill out all fields before continuing! ☺︎')
+    } else {
+      currentCover = createCover(userCover.value, userTitle.value, userDecr1.value, userDecr2.value);
+      fullCover.innerHTML = `<img class="cover-image" src="${currentCover.coverImg}">
+        <h2 class="cover-title">${currentCover.title}</h2>
+        <h3 class="tagline">A tale of <span class="tagline-1">${currentCover.tagline1}</span> and <span class="tagline-2">${currentCover.tagline2}</span></h3>
+        <img class="price-tag" src="./assets/price.png">
+        <img class="overlay" src="./assets/overlay.png">`;
```

Comment:
totally non-blocking, but shoot I'm not sure I like this giant `innerHTML` with all of the tags/classes over what we had before above.  But I cant't find anything googling that points to any best practices on this!

Reply:
Yeah no problem! I thought I had a comment somewhere explaining why I changed it but it looks like I forgot. Basically, when multiple new books were made, it wasn't showing on the home screen, but it was saving. So I used `.innerHTML` to grab from the HTML code exactly to get the book to show.

Reply:
![Screenshot 2023-04-02 at 11 55 35 AM](https://user-images.githubusercontent.com/126219151/229364277-69557e07-23cc-4ff8-9d16-769478c64e65.png)
Ahhh I made the comment on the other PR lol